apm-sniffer/apm-sdk-plugin/postgresql-8.x-plugin/pom.xml

```diff
@@ -48,6 +49,11 @@
             <version>${project.version}</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <version>${guava.version}</version>
+        </dependency>
     </dependencies>
```

Comment:
You **MUST NOT** use third-party dependencies in agent plugin

Reply:
Why need this lib? BTW. You are just logging parameter into the span, seems not a complex work. And clearly, the MySQL plugin has given a demo about how to do it.